env/GAEAC5.env

```diff
@@ -175,6 +175,14 @@ case ${step} in
     export NTHREADS_CYCLE=${threads_per_task:-14}
     [[ ${NTHREADS_CYCLE} -gt ${max_tasks_per_node} ]] && export NTHREADS_CYCLE=${max_tasks_per_node}
     export APRUN_CYCLE="${APRUN_default} --cpus-per-task=${NTHREADS_CYCLE}"
+
+    ntdiv6=$((ntasks/6))
+    ntasks_regrid=$((ntdiv6*6))
```

Comment:
is `ntdiv6` used anywhere? why do:
`ntasks_regrid = ntasks/6 * 6` ???

Reply:
ESMF requires that the number of tasks be divisible by 6. Since bash can only handle integers, I assume this is a safe way to do that. 

I'll add a comment to explain why we're doing this.